### ISSUE TITLE
DP#112: Artistrict - The biggest casting show in history.

### ISSUE THREAD

**Issue by timinho111** [district0x/district-proposals#112]
# Name: **Artistrict** - The biggest casting show in history.

## Purpose 

Through the huge audience over the internet and the voting system of district0x we can couple two amazing things.

In the last few years some successful artists (mostly musicians) have become popular through the snowball effects of the internet. Most of them have contracts by some huge companies where only a few people decide the direction of not only the firm but also the artists career.

Now think about the chance district0x is bringing to us:
How about a plattform where everyone can submit an audio audience (with video) and _stakeholders_ (via DNT) as well as _users_ can **vote** for the most talented ones?

## Description
Lets go through the third part of the purpose step-by-step:

In the first run through it is going to start out with only musicians. After exactly 6 months after launch the **first season** is over. The next 3 months is what I call the political break. In this **political break** all stakeholders will vote on topics submitted by _stakeholders_ via _GitHub_ during the first season.
This could be the length for example, The price pool (which I will come later, too) and everything else which they like to discuss (through **popular vote**). Another topic could be the implementation of other forms of art (like dancing, acting (e-)sports and so on). 

The second part is about **voting**. The _stakeholders_ should have a noticable advantage in voting rights. _Stakeholder_ **votes** should count for about 10-1000 times per 0.1 DNT compared to _users._

The winner of the season gets the contributed price pool in DNT. I am not sure how many people would donate to this idea, but if it gets implemented I would donate 50.000 DNT (about 10000$ with the price right now) for the price pool of season 1 (if that is possible).

I would conclude with another scenario at the first **political break**.
Think about the possibilty that the plattform itself could transfer very much by itself, too. What if we make it possible through the **first season**, that this project gets so much attention, that a lot of people are willing to contribute on their own.
Then it might not be even an option to do it seasonally but rather as a huge internet music label (or in future actor agent)?

**Comment by mckmuze** [comment]
There is a lot here that needs to be dug into. I'm a musician and play with these concepts in association with many other projects as well. 

What I see eventually happening is a distributed artist development platform is what it boils down to every time. That is essentially what a label is. 

They sign talent, develop them and extract wealth from them like cattle. In a distributed model an artist has more say, freedom to pursue and freedom to fail. Artists typically have to fake their death just to escape a contract in the traditional industry and have no meaningful connection to their fans beyond money. It's sad really.  

I wrote the #64 proposal with this type of artist development in mind. I encourage you to develop this idea more, I would love to collaborate.

**Comment by timinho111** [comment]
Yes, I just wrote down some quick thoughts. I am planning to go a little more into detail for those two options in the next couple of days. I have to clear paths in mind and I was not even kidding.
Wouldn't it be interesting to see what the people think (per popular vote) how the district ("Casting Show" aka one guy/girl/group takes it all vs countinious support of several artists? That is what I find really fascinating about district0x. 
Please keep in mind that I do not want to turn the "Casting Show" into a "S*** Show" ;)

**Comment by timinho111** [comment]
ETH adress: 0x28f808b3F16Be599d6d5fd01c58A12B6f7eBA8a6

Slack name: bskkkkn

**Comment by Bradymck** [comment]
Hey @timinho111 

I'm not sure if you're aware but we recently launched a new bounty to migrate your proposals to the actual District Registry: https://registry.district0x.io/

We're replacing the old voting app with the registry. Let me know if you need help but I would love to see you migrate this over so you can claim your DNT.

It does take a 10,000 DNT submit to submit your proposal but this gives you an extra 2000 you can stake in the registry beyond the deposit amount.

Hit me up on Telegram or Discord if you need help or have questions.

Telegram: https://t.me/district0x/75217
Discord: https://discord.gg/P9RQejv

PS, please excuse the canned response. I am encouraging everyone here to start migrating so they can claim their 12000 DNT.
